readme.md

```diff
@@ -12,3 +12,39 @@ libqmlbind is a C library for creating QML bindings for other languages easily t
   * High portability
 * Build with qmake
   * No need to configure compilation flags for Qt manually
+
+## How to build
+### Requirements
+* A C++11-capable compiler: libqmlbind is tested against `clang 3.4`.
+* Qt5: libqmlbind is tested against `Qt 5.5.1`. Everything `>= 5.2` might work, too.
+
+### Build the Code
+First, clone the repository with all its submodules:
+```
+> git clone --recursive git@github.com:seanchas116/libqmlbind.git
+```
+
+Alternatively, if you already cloned but forgot `--recursive`, use this inside `libqmlbind`:
+```
+> git submodule update --init --recursive
+```
+
+Use qmake to generate the makefile. The `-r` option is needed because `libqmlbind` depends on Qt's private headers.
+```
+> # if you want to use your default compiler:
+> qmake -r
+> # if you want to use clang:
+> qmake -r QMAKE_CC=clang QMAKE_CXX=clang
+```
+
+and then build it, `-j$(nproc)` uses as many threads as you have CPU cores:
+```
+> make -j$(nproc)
```

Comment:
Oh, I didn't know `nproc` is not part of MacOS.

Just thought a hint for `-j` was a nice addition, as it reduced libqmlbind`s initial compile time to under one minute for me, and it seems like the option is not widely known: I was able to solve many complaints about slow compiling in another project by proposing to use the `-j` option with the number of logical cores available.